source/assets/scripts/main.js

```diff
@@ -95,6 +95,7 @@ function taskCount() {
 
 /**
  * load db from local storage
+ * 
  * @returns {Array<Object>} An array of users found in localStorage
```

Comment:
Technically it's array of user object that contains their username, password, and tasks inside